source/connect/connect-from-the-command-line/command-line-options.txt

```diff
@@ -114,8 +114,7 @@ value in the :guilabel:`Settings` panel.
 
 .. option:: --protectConnectionStrings
 
-   Hide credentials in connection strings. Passwords in connection 
-   strings are displayed as ``*****``. 
+   .. include:: /includes/fact-protect-connection-strings.rst  
```

Comment:
Maybe also provide a link to the new task page in this description? https://docs-mongodbcom-staging.corp.mongodb.com/compass/docsworker-xlarge/DOCSP-26281-protectconnectionstrings-option/config-file/protect-connection-strings/

Reply:
In this case, would it make sense to link the corresponding task page (if available) to each CLI option (i.e. for `networkTraffic` and `forceConnectionOptions`

Reply:
Got it. I think since there aren't too many to add, I've added them along in this ticket. I do know this will be revisited during the IA changes phase of this epic, so can circle back when we get to that point.